projects/plugins/jetpack/class.jetpack-gutenberg.php

```diff
@@ -699,6 +699,56 @@ public static function enqueue_block_editor_assets() {
 		);
 	}
 
+	/**
+	 * Add the Gutenberg editor stylesheet to iframed editors, such as the site editor,
+	 * which don't have access to stylesheets added with `wp_enqueue_style`.
+	 *
+	 * This workaround is only supposed to be used by WordPress.com Simple sites.
+	 *
+	 * @since $$next-version$$
+	 *
+	 * @return void
+	 */
+	public static function add_iframed_editor_style() {
+		if ( ! self::should_load() ) {
+			return;
+		}
+
+		global $pagenow;
+		$allowed_pages       = array( 'admin.php', 'themes.php' );
```

Comment:
Note that the site editor url can also be /wp-admin/site-editor.php in WP 5.9 without Gutenberg active, so we should probably account for that as well.

Reply:
TIL!